CHANGELOG.md

```diff
@@ -58,6 +58,9 @@ The following changes only apply when using `sentry-cli` via the npm package [`@
 
 - Fixed misleading error message claiming the server doesn't support chunk uploading when the actual error was a non-existent organization ([#2930](https://github.com/getsentry/sentry-cli/pull/2930)).
 
+### Internal changes
+
+- Removed `node-fetch` dependency when using the NPM package ([#2993](https://github.com/getsentry/sentry-cli/pull/2993))
```

Comment:
Since this is an internal change, we should not include it in the changelog imo. 

```suggestion
```

I would also update the PR title from `feat:` to `ref:`, as it is not really a new feature from the user perspective. And, that way, the DangerJS action will not complain about a missing changelog entry, since changelog entries are not required for `ref:` PRs.

Reply:
I removed it and renamed the PR